doc/README.md

```diff
@@ -584,3 +584,31 @@ The system performs the fee validation and returns a result indicating that the
 - The `X-SkipFeeValidation` header takes precedence over `X-CumulativeFeeValidation` and causes the fee validation to be skipped.
 
 This feature is crucial to ensure that the chain of unmined transactions has sufficient fees, which is essential for the effective management of the transaction network.
+
+## Strict cumulative fees validation
+
+The fature is extention of the "Cumulative Fee Validation". It adds checking the lenght of unmined ancestors chain to validation.
```

Comment:
There are some type-os in this line.

"This **feature** is **an** exten**s**ion of the "Cumulative Fee Validation". It adds checking the leng**th** of unmined ancestors chain to validation."


We should add a bit more explanation to that. So I would suggest this
"It adds checking the length of unmined ancestors chain against the setting of child pays for parent limitation `limitcpfpgroupmemberscount` of the connected nodes."

Reply:
Changed